samples/CallWithChat/src/app/views/CallScreen.tsx

```diff
@@ -36,6 +36,16 @@ export const CallScreen = (props: CallScreenProps): JSX.Element => {
     locator,
     /* @conditional-compile-remove(PSTN-calls) */ alternateCallerId
   } = props;
+
+  // Disables pull down to refresh. Prevents accidental page refresh when scrolling through chat messages
+  // Another alternative: set body style touch-action to 'none'. Achieves same result.
+  useEffect(() => {
+    document.body.style.overflow = 'hidden';
```

Comment:
does this disable pulldown behavior because there is no overflow on the page?

Reply:
Yes, essentially on safari ios there is no explicit option to disable pull down to refresh. 
The intermediate solution is to either apply a 
overflow: 'hidden' or a touch-action: 'hidden' to the body element to prevent a pull-down from occurring at all.

Reply:
I did not apply this fix to the composites and have only applied the fix to the samples as this can 100% affect Contosos use cases if integrated as a modal, iframe, etc..

I added instructions into storybook for best practices for mobile.